tensorflow/stream_executor/rocm/rocm_driver.cc

```diff
@@ -486,15 +486,8 @@ GpuDriver::ContextGetSharedMemConfig(GpuContext* context) {
                                                      size_t uint32_count) {
   ScopedActivateContext activation{context};
   void* pointer = absl::bit_cast<void*>(location);
-  unsigned char valueC = static_cast<unsigned char>(value);
-  uint32_t value32 = (valueC << 24) | (valueC << 16) | (valueC << 8) | (valueC);
```

Comment:
wont this logic now be needed in the call to hipMemset in the SynchronousMemsetUint8 routine (line 450 above)?

Reply:
https://github.com/ROCm-Developer-Tools/HIP/blob/master/include/hip/hcc_detail/hip_runtime_api.h#L1494

this line is correct.

Reply:
and for the call site of U8, probably need to switch to:
https://github.com/ROCm-Developer-Tools/HIP/blob/master/include/hip/hcc_detail/hip_runtime_api.h#L1505

Reply:
yes, switching the call at line 450 to hipMemsetD8 would work too.